src/main/java/com/jfrog/ide/idea/inspections/JFrogSecurityAnnotator.java

```diff
@@ -37,19 +39,31 @@ public PsiFile collectInformation(@NotNull PsiFile file, @NotNull Editor editor,
 
     @Nullable
     @Override
-    public List<JFrogSecurityWarning> doAnnotate(PsiFile file) {
-        ScanManager scanManager = ScanManagersFactory.getScanManagers(file.getProject()).stream()
-                .findAny()
-                .orElse(null);
-        return scanManager != null ? scanManager.getSourceCodeScanResults() : new ArrayList<>();
+    public List<ApplicableIssueNode> doAnnotate(PsiFile file) {
+        List<ApplicableIssueNode> applicableIssues = new ArrayList<>();
+        ComponentsTree componentsTree = LocalComponentsTree.getInstance(file.getProject());
+        if (componentsTree == null || componentsTree.getModel() == null) {
+            return null;
+        }
+        Enumeration<TreeNode> roots = ((BaseTreeNode) componentsTree.getModel().getRoot()).children();
+        for (TreeNode root : Collections.list(roots)) {
```

Comment:
I'd prefer using the enumeration directly and not initializaing a new list. The new list duplicating the children of the tree and causes some redundant work.
Same for the for loop below.